plugins/base/src/test/kotlin/resourceLinks/ResourceLinksTest.kt

```diff
@@ -184,7 +184,7 @@ class ResourceLinksTest : BaseAbstractTest() {
                     assertNull(writerPlugin.writer.contents["scripts/relativePath.js"])
                     assertNull(writerPlugin.writer.contents["styles/relativePath.js"])
                     Jsoup
-                        .parse(writerPlugin.writer.contents["root/example.html"])
+                        .parse(writerPlugin.writer.contents.getValue("root/example.html"))
```

Comment:
it does not matter but above there is `[]`

Reply:
Yeah, but it doesn't lead to an error. On the contrary, it's quite logical. `map[key]` may return null, so above it's evaluated to `assertNull(null)`

Whereas `Jsoup.parse` accepts only `@NotNull`, thus `.getValue()`, which throws an error